db.js

```diff
@@ -0,0 +1,13 @@
+
+
+const Pool = require("pg").Pool;
+
+const pool = new Pool({
+    user: "postgres",
+    password: "admin",
+    database: "auction",
+    host: "localhost",
```

Comment:
Host and port should be variables. What if you deploy this application in the future?